third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -393,6 +397,44 @@ func resourceComposerEnvironment() *schema.Resource {
 								},
 							},
 						},
+<% end -%>
+<% unless version == "ga" -%>
+            "database_config": {
```

Comment:
nit: mind fixing the spacing on this line & `web_server_config`'s line? They're spaced as spaces instead of tabs.

Reply:
Sure.